rules-tests/CodingStyle/Rector/Enum_/EnumCaseToPascalCaseRector/config/configured_rule.php

```diff
@@ -6,4 +6,5 @@
 use Rector\Config\RectorConfig;
 
 return RectorConfig::configure()
-    ->withRules([EnumCaseToPascalCaseRector::class]);
+    ->withRules([EnumCaseToPascalCaseRector::class])
```

Comment:
Like other your PR, Please don't change existing test config, use different class & config on purpose, with use of dynamic provider.

Reply:
In this case there's really no need for a separate config and test case just to test one fixture

Reply:
If you really insist then i can keep both, both the autoload fixture will have to be moved to a separate directory because the test cases were conflicting

Reply:
No, that on purpose :), see the whole discussion on original PRs :)

- https://github.com/rectorphp/rector-src/pull/6890
- https://github.com/rectorphp/rector-src/pull/6899